### ISSUE TITLE
Removing rooms from communities is hard to find

### ISSUE THREAD

**Issue by 5chdn** [vector-im/element-web#5743]
### Description

This is critical in combination with #5742 for us.

One can not remove channels from communities.

### Steps to reproduce

- Add a channel to the community
- Try to remove it
- Find this functionality is missing

Describe how what happens differs from what you expected.

Log: not sent

![screenshot at 2017-11-30 12-16-54](https://user-images.githubusercontent.com/15729797/33428293-871c372a-d5c8-11e7-9fae-9b3f805b8400.png)

### Version information

- **Platform**: web in-browser and web in-rambox.

For the web app:

- **Browser**: Chrome 62, and Firefox 57?
- **OS**: Arch Linux
- **URL**: riot.im/app

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/33429467-de67c034-d5c4-11e7-8bd0-911c2b9fcd8a.png)

its in the right panel...

**Comment by turt2live** [comment]
This should be re-opened as this is the no. 1 complaint about removing rooms. It doesn't make any sense to have the remove button on the other side of the app from the add button, just put them together.

**Comment by 5chdn** [comment]
@t3chguy I don't have this button, no admin tools at all.

![screenshot at 2017-11-30 12-10-20](https://user-images.githubusercontent.com/15729797/33428026-a222549c-d5c7-11e7-8711-15effaddc016.png)

**Comment by t3chguy** [comment]
yeah for those still wondering, click the `#` in the right panel, then click the room and then you get Admin Tools

**Comment by lampholder** [comment]
In the first draft, all of the UX for manipulating rooms'/users' membership of a group was on the RHS.

But you've discovered as we did that nobody can see the # icon on the top right there 😞 Adding groups was added to the middle as a quick optimisation; we should certainly extend that by adding a more discoverable way to remove groups.

**Comment by turt2live** [comment]
Duplicate of https://github.com/vector-im/riot-web/issues/5529 (although this issue has a bit more information)

**Comment by eaterjolly** [comment]
I would like to express my opinion UX should involve some difficulties/challenges. 
In this case, the page layout follows a certain logic, so solving the difficulty does give satisfaction.

**Comment by fti7** [comment]
Wow, i've searched 15min in the Client! Not found. Thanks to this Issue.
Please improve that! :-)

**Comment by KarlWolters92** [comment]
I have wasted 1h of searching. I created a room in riot using the Mac app. I just want to delete that room again. How can I do that? I looked also in the browser version. Under admind tools I just find mute, kick, ban and remove recent messages....

**Comment by z3ntu** [comment]
@KarlWolters92 If you're in the main screen of the community, you have the member list on the right and on top of that there's a "Members" button (icon: person) and a "Rooms" button (icon: #). You click on the "Rooms" one, select your room in the list on the right and press "Remove from community". https://github.com/vector-im/riot-web/issues/5743#issuecomment-348166906 has a screenshot of it

**Comment by KarlWolters92** [comment]
@z3ntu thank you for helping me out! I do not have that rooms button. Only the persons icon that shows me the members of any particular room that I am in. If I click in the lower left corner on communities, it says I am currently not belonging to any communities.

To clarify: I have been invited into a room by my a colleague. This collegue invited all member of an institution. Then I created a room and invited only a subgroup of the institution involved in a particular project. Now I wanna delete this room.

**Comment by aaronraimist** [comment]
@KarlWolters92 You want #6978 then. This issue is talking about removing rooms from communities, not deleting rooms.

TLDR on #6978 is: You cannot currently delete rooms, if everyone leaves a room then it will become unjoinable and the server can clean it up.

**Comment by ineiti** [comment]
I searched for 30 minutes, found it. One week later I searched for 10 minutes, googled it, found this issue, searched for 5 minutes with this issue, and found it.

There _is_ some issue with this. But, well, the whole thing of communities is non-intuitive to me.

**Comment by johegl** [comment]
I have just come to Riot.im/Matrix and found this absolutely unintuitive. Until I found this GitHub issue here, I had no clue what to do.
OT: Adding a room was also not intuitve. You create a community, click the plus in the rooms section and it creates a room outside your community... you need to add it to your community from its settings after that. This is going to put off lots of non-tech users and such that just want to work efficiently :-(

**Comment by t3chguy** [comment]
This is fixed by Spaces landing in Beta in next Release
